### PR TITLE
Add more messages to identify the problem when installing packages

### PR DESCRIPTION
**Pull request by anatoly-os**
Show message while unpacking and loading sfzs

### CODE REVIEW

File: mscore/musescore.cpp [hunk 1]
```diff
@@ -496,11 +507,16 @@ bool MuseScore::importExtension(QString path)
...
             }
       zipFile3.close();
+      msgBox->hide();
 
       mscore->reloadInstrumentTemplates();
       mscore->updateNewWizard();
       mscore->updateInstrumentDialog();
 
+      msgBox->setWindowTitle(tr("Packages installer"));
+      msgBox->setText(tr("<p align='center'>Please wait, loading soundfonts... </p>"));
```
Comment (Jojo-Schmitz) [inline mscore/musescore.cpp]: Can we separate the HTML from the text to translate here? Like
`msgBox->setText("<p align='center'>%1</p>").arg(tr("Please wait, loading soundfonts..."));``

File: mscore/musescore.cpp [hunk 2]
```diff
@@ -487,6 +487,17 @@ bool MuseScore::importExtension(QString path)
...
             }
+
+      QMessageBox* msgBox = new QMessageBox(parent);
+      msgBox->setWindowModality(Qt::ApplicationModal);
+      msgBox->setWindowFlags(Qt::WindowFlags(Qt::Dialog | Qt::FramelessWindowHint | Qt::WindowTitleHint));
+      msgBox->setTextFormat(Qt::RichText);
+      msgBox->setMinimumSize(300, 100);
+      msgBox->setMaximumSize(300, 100);
+      msgBox->setStandardButtons(0);
+      msgBox->setText(tr("<p align='center'>Please wait, unpacking extension...</p>"));
```
Comment (Jojo-Schmitz) [inline mscore/musescore.cpp]: same here (see below)

File: mscore/musescore.cpp (hunk 1 above)
Comment (anatoly-os) [reply mscore/musescore.cpp]: ah, yes, thank you!

File: mscore/musescore.cpp [hunk 3]
```diff
@@ -496,11 +507,16 @@ bool MuseScore::importExtension(QString path)
             return false;
             }
       zipFile3.close();
+      msgBox->hide();
 
       mscore->reloadInstrumentTemplates();
       mscore->updateNewWizard();
       mscore->updateInstrumentDialog();
 
+      msgBox->setWindowTitle(tr("Packages installer"));
```
Comment (Jojo-Schmitz) [inline mscore/musescore.cpp]: Title Case: `Packages Installer"

File: mscore/resourceManager.cpp [hunk 4]
```diff
@@ -302,26 +302,37 @@ void ResourceManager::downloadLanguage()
...
+      bool verifyFileRes = saveFileRes && verifyFile(localPath, hash);
+      if(!verifyFileRes) {
             QFile::remove(localPath);
             button->setText(tr("Failed, try again"));
             button->setEnabled(true);
+
+            QMessageBox msgBox;
+            msgBox.setStandardButtons(QMessageBox::Ok);
+            msgBox.setTextFormat(Qt::RichText);
+            msgBox.setWindowTitle(tr("Extensions installation failed"));
```
Comment (Jojo-Schmitz) [inline mscore/resourceManager.cpp]: I guess here it too should be Title Case, `Extensions Installation Failed`

**Comment by Jojo-Schmitz** [comment]
while at it, please fix the string at mscore/musescore.cpp line 395, add a space between : and %

**Comment by MarcSabatella** [comment]
Not sure if I'm the only one experiencing the instant failure installing the MDL extension, but as discussed on IRC, I was unable to download the MDL extension while at the local library using their public internet.  It failed repeatedly right after trying to create the Tempmdl.muxt file but before actually downloading anything.  But worked instantly when I tried from home later.  So our working theory is that it had to do with firewall settings or some such.

I won't be able to get to the library to try again today but might in next few days.  Maybe these messages will help confirm what actually happened.  Also, I do now have my own 2.3 build, so I can also try that under the debugger and hopefully get more info.

**Comment by anatoly-os** [comment]
Marc, I've tested the case in Airplane mode on my laptop. The symptoms were the same. Instant finishing of downloading process and Failed result. I've added the message with "Check internet connection" text. I'm sure, the library has proxy limitation for big files (e.g .larger than 10Mb). It was not about languages, cause language packs are small.

**Comment by Jojo-Schmitz** [comment]
http vs https?

**Comment by MarcSabatella** [comment]
Makes sense.  Will discuss other bugs on IRC :-)

**Comment by anatoly-os** [comment]
Jojo, we use http for both languages and extensions, but languages worked.

**Pull request merged**